### PR TITLE
Prepare for 1.2.0 release

### PR DESCRIPTION
**Pull request by echoromeo**
@capelevy, @oyvindwe it makes sense to say 2.0.0 and not 1.3.0 here right?

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,7 +50,7 @@
 
     # This should be your name or the name of the organization which owns the
     # project.
-    author='echoromeo, capelevy',
+    author='echoromeo, capelevy, oyvindwe',
     author_email='felis.lynx@gmail.com',  # Optional
```
Comment (oyvindwe) [inline setup.py]: Remove `author_email`?

**Review by oyvindwe — DISMISSED** [review]

**Comment by oyvindwe** [comment]
I bumped to 1.2.0 in my PR since it is backwards compatible with 1.1.x.

See https://semver.org/

**Comment by echoromeo** [comment]
Oh, I missed your bump to 1.2.0. 

> it is backwards compatible with 1.1.x.

I might have missed something here, but I believed that ```await hub.start()``` alone removes backwards compatibility?

**Comment by oyvindwe** [comment]
If you call `nobo()` without passing in `loop`, it will start in the background as 1.1.x. Also, it is optionally to register the callback. See `README.md`: https://github.com/echoromeo/pynobo#backwards-compatability

Note I made a typo in that heading - it should read "Backwards compatibility".

**Review by oyvindwe — APPROVED** [review]

**Comment by oyvindwe** [comment]
Thank you @echoromeo !

**Pull request merged**